release_radar.js

```diff
@@ -386,13 +392,27 @@ function ajax_load(path) {
 
 
 function get_cache() {
```

Comment:
I dont use GM_ functions because the desktop application obviously does not support them and I want to keep the desktop and userscript version as similar as possible. In my experience, the localstorage gets cleared basically never, even when logging out so I dont see a reason to swtich to GM_

Reply:
This just need a few extra `if( GM_stuff !== undefined)` to work as is on desktop : I left everything in place for localstorage support for users to migrate their current config.
On my side, using these functions should allow synchronisation of settings and cache between devices/browsers, which is a use case I encounter (I'm using ViolentMonkey on Firefox BTW, on multiple computers using Deezer).
And this would allow for a way of backing up the config without exporting/importing a JSON through a custom UI.

Reply:
I wont implement the GM_ stuff, you'll have to use your forked version in the future for that, sorry. I will probably just implement a config exporting/importing logic. Im currently working on hijacking the deezer settings for cloud savings anyways.

I will look through your changes and implement some. This PR wont get merged (I will implement them in the next release manually (credit you obv)), because I disagree with a few changes, I will give more details in comments later.

Reply:
> Im currently working on hijacking the deezer settings for cloud savings anyways.

That's an interesting way of doing it !

> I will look through your changes and implement some. This PR wont get merged (I will implement them in the next release manually (credit you obv)), because I disagree with a few changes, I will give more details in comments later.

My work is merely suggestions, pick whatever you like as is or reimplement it all together. I'll just open a PR to "ping" you that I've worked on something you should consider, but close them as you like.

> you'll have to use your forked version in the future for that, sorry.

I was mulling over it, so now I have a good reason to do so :)

Reply:
keep the suggestions coming :D